example/src/main.rs

```diff
@@ -0,0 +1,33 @@
+extern crate civet;
+extern crate conduit;
+extern crate conduit_router;
+
+use std::collections::HashMap;
+use std::io::{self, Cursor};
+use std::sync::mpsc::channel;
+
+use civet::{Config, response, Server};
+use conduit::{Request, Response};
+use conduit_router::{RouteBuilder, params};
+
+
+fn name(req: &mut Request) -> io::Result<Response> {
+    let name = params(req).find("name").unwrap();
+    let bytes = format!("Hello {}!", name).into_bytes();
+
+    Ok(response(200, HashMap::new(), Cursor::new(bytes)))
+}
+
+
+fn main() {
+    let mut router = RouteBuilder::new();
+
+    router.get("/:name", name);
+
+    let router = router;
```

Comment:
Yes, it can be omitted. I think I've seen something similar in the source of the crates.io (but I can't find it right now) and it's seemed reasonable to 'freeze' the routes. Was it a bad idea? 

I'll remove it because it can be misleading in an example application.